### PR TITLE
Bug 1221439 - [TV Browser] Missing the "x" button to clean the passwo…

### PR DESCRIPTION
**Pull request by begeeben**
…rd in Firefox Account - Sign In page

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=78cb550f68f87b9fcf82ecbf5ee9ea46d6f08cb2

File: tv_apps/smart-system/fxa/js/screens/fxam_refresh_auth.js [hunk 1]
```diff
@@ -84,13 +84,29 @@ var FxaModuleRefreshAuth = (function() {
       this.importElements(
         'fxa-pw-input-refresh',
         'fxa-show-pw-refresh',
-        'fxa-forgot-password-refresh'
+        'fxa-forgot-password-refresh',
+        'fxa-pw-clean-btn'
       );
 
       this.fxaPwInputRefresh.addEventListener('input', (function(event) {
```
Comment (weilonge) [inline tv_apps/smart-system/fxa/js/screens/fxam_refresh_auth.js]: If it is an anonymous function, can we use arrow function here? I see there are different function style in this file. That would be great to make them consistent and avoid using ```bind```.

File: tv_apps/smart-system/fxa/js/screens/fxam_enter_password.js [hunk 2]
```diff
@@ -104,6 +110,16 @@ var FxaModuleEnterPassword = (function() {
         setTimeout(this.fxaPwInput.select.bind(this.fxaPwInput));
       });
 
+      this.fxaPwCleanBtn.addEventListener(
+        'click',
+        function onCleanButtonClick(e) {
```
Comment (weilonge) [inline tv_apps/smart-system/fxa/js/screens/fxam_enter_password.js]: How about using arrow-function style?

File: tv_apps/smart-system/fxa/js/screens/fxam_refresh_auth.js [hunk 3]
```diff
@@ -84,13 +84,29 @@ var FxaModuleRefreshAuth = (function() {
...
+          this.fxaPwCleanBtn.classList.add('show');
+        } else {
+          this.fxaPwCleanBtn.classList.remove('show');
+        }
         _enableDone(event.target);
       }).bind(this));
 
+      this.fxaPwCleanBtn.addEventListener(
+        'click',
+        function onCleanButtonClick(e) {
```
Comment (weilonge) [inline tv_apps/smart-system/fxa/js/screens/fxam_refresh_auth.js]: How about using arrow-function style?

File: tv_apps/smart-system/fxa/js/screens/fxam_enter_password.js [hunk 4]
```diff
@@ -79,14 +79,20 @@ var FxaModuleEnterPassword = (function() {
...
         'fxa-pw-input',
         'fxa-show-pw',
-        'fxa-forgot-password'
+        'fxa-forgot-password',
+        'fxa-pw-clean-btn'
       );
       // Add listeners
       this.fxaPwInput.addEventListener(
         'input',
         function onInput(event) {
```
Comment (weilonge) [inline tv_apps/smart-system/fxa/js/screens/fxam_enter_password.js]: How about using arrow-function style?

**Pull request merged**